spec/controllers/static_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# spec/controllers/static_controller_spec.rb
+
+require "rails_helper"
+
+RSpec.describe StaticController, type: :request do
+  describe "#simulatorvue" do
+    it "renders the simulatorvue template" do
+      get "/simulatorvue"
+
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include(File.read(Rails.public_path.join("simulatorvue", "index.html")))
+    end
+  end
+end
```

Comment:
One solution is working fine, in all other cases undercover is raising the issue

In the `rails_helper` file, we need to put this line
```ruby
  config.include Devise::Test::ControllerHelpers, type: :controller
```

And the updated specs will be
```ruby
# frozen_string_literal: true

# spec/controllers/static_controller_spec.rb

require "rails_helper"

RSpec.describe StaticController, type: :controller do
  describe '#simulatorvue' do
    it 'renders the simulatorvue template' do
      get :simulatorvue, params: {path: "/"}

      expect(response).to have_http_status(:success)
      expect(response.body).to include(File.read(Rails.public_path.join("simulatorvue", "index.html")))
    end
  end
end
```

Reply:
Thanks for your help review dog issue is fixed now.